units/ArmBuildings/LandDefenceOffence/armamb.lua

```diff
@@ -210,19 +210,17 @@ return {
 		},
 		weapons = {
 			[1] = {
-				badtargetcategory = "VTOL GROUNDSCOUT",
```

Comment:
removal of vtol sounds like unrelated cleanup so could be in its own tiny pr

Reply:
Yes, though in this case I mostly care about not bundling unrelated crap into the same PR. Best to keep changes small and focused. I guess it's already small and the line was being touched anyway but my first thought was that VTOL is not a subset of SURFACERAIDER so this would've been a logic change if not for the onlytargetcategory not allowing VTOL anyway.